ports/rejuvenation/rejuvenation/preset/mkxp.json

```
@@ -0,0 +1,531 @@
+
+
+    // Index all accesible assets via their lower case path
+    // (emulates windows case insensitivity)
+    // (default: enabled)
+    //
+    // "pathCache": true,
+
+    // Add 'rtp1', 'rtp2.zip' and 'game.rgssad' to the asset search path
+    // (multiple allowed). You can use folders, RGSS archives, and any archive
+    // formats supported by PhysicsFS; see the compatibility list at:
+    // https://www.icculus.org/physfs/docs/html/
+    // (default: none)
+    //
+    // "RTP": [
+    //     "/path/to/rtp1",
+    //     "/path/to/rtp2.zip",
+    //     "/path/to/game.rgssad",
+    // ],
+
+    // Similar to the RTP option, except items are loaded before
+    // the game archive and folder, for incremental game updates
+    // or modding.
+    // (default: none)
+    //
+    // "patches": [
+    //     "/path/to/patch1.zip",
+    //     "/path/to/patch2",
+    // ],
+
+
+    // Use the script's name as filename in warnings and error messages
+    // (default: enabled)
+    //
+    // "useScriptNames": true,
+
+
+    // Font substitutions allow drop-in replacements of fonts
+    // to be used without changing the RGSS scripts,
+    // eg. providing 'Open Sans' when the game thinkgs it's
+    // using 'Arial'. Font family to be substituted and
+    // replacement family are separated by one sole '>'.
+    // Be careful not to include any spaces.
+    // This is not connected to the built-in font, which is
+    // always used when a non-existing font family is
+    // requested by RGSS.
+    // (default: none)
+    //
+    // "fontSub": [
+    //     "Arial>Open Sans",
+    //     "Times New Roman>Liberation Serif",
+    // ],
+
+
+    // Because mkxp is usually distributed as a stand alone
+    // build, no predefined load paths are initialized
+    // ($:, $LOAD_PATH) in the MRI backend. With this option,
+    // they can be specified manually (eg. when using a system
+    // libruby.so). It is however recommended to statically
+    // link all required gems into libruby.so.
+    // (default: none)
+    //
+    // "rubyLoadpath": [
+    //     "/usr/lib64/ruby/",
+    //     "/usr/local/share/ruby/site_ruby",
+    // ],
+
+    // Determines whether MJIT is enabled. This probably
+    // won't work unless you also have the header file
+    // that it needs. Only works with Ruby 2.6 or higher.
+    // This Ruby feature is experimental.
+    // (default: false)
+    //
+     "JITEnable": true,
+    
+    // Determines what level of verbosity to use when
+    // logging MJIT events. Starts at 0, which is next
+    // to nothing. Set it higher to see more.
+    // (default: 0)
+    //
+    // "JITVerboseLevel": 0,
+    
+    // Determines how many compiled methods that Ruby
+    // will keep in its MJIT cache.
+    // (default: 100)
+    //
+    // "JITMaxCache": 100,
+    
+    // Determines how many times a function has to be
+    // called before it is compiled by MJIT.
+    // (default: 10000)
+    //
+     "JITMinCalls": 5000,
```

Same as in reborn, it seems to be the last entry.
```suggestion
     "JITMinCalls": 5000
```